package.json

```diff
@@ -160,6 +159,7 @@
     "react-autosuggest": "^9.3.2",
     "react-docgen": "^3.0.0-beta9",
     "react-dom": "^16.1.1",
+    "react-flow-types": "^0.2.0-beta.6",
```

Comment:
@rsolomon these need to be in dependencies instead of dev so that shadow files work for downstream usage.

Regardless, there is no impact to having them in dependencies as they are stripped out for browser use.